### ISSUE TITLE
[Bug]: Wokingham Borough Council fails to load dates

### ISSUE THREAD

**Issue by MichaelMKKelly** [mampfes/hacs_waste_collection_schedule#2488]
### I Have A Problem With:

A specific source

### What's Your Problem

install this recently so limited experience: it worked on initial setup then failed. when I reinstalled it worked again for a week but then failed to get new dates. now does not work at all with the error "The source returned an invalid response: "'NoneType' object has no attribute 'find_all'". Please check the provided arguments and try again."

appears to be having trouble reading the council webpage which does appear to be working and "form-item__dropdown" does appear to exist but maybe something has changed and soup is having issues with it. I have no experience with soup so i am reaching my limit in self troubleshooting for now. I may try to keep digging if I get some time but perhaps someone with more experience could have a look at this source and see if they see the same issue?

if i try adding the property value i get the error
"The source returned an empty response. Please check the provided arguments and try again."
instead

### Source (if relevant)

wokingham_gov_uk

### Logs

```Shell
2024-08-17 01:54:48.673 ERROR (SyncWorker_24) [waste_collection_schedule.source_shell] fetch failed for source Wokingham Borough Council:
Traceback (most recent call last):
  File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 145, in fetch
    entries = self._source.fetch()
              ^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/wokingham_gov_uk.py", line 79, in fetch
    addresses = dropdown.find_all("option")
                ^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'find_all'
```

### Relevant Configuration

```YAML
as aboove, i have tried with and all combinations of using property/adddress
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by 5ila5** [comment]
This was already fixed in #2395

This will be part of the next release or is available now as master (git) version (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)

**Comment by MichaelMKKelly** [comment]
thanks, sorry I somehow totally missed that PR when checking if it was already reported. It is all working now though.
